hpx/parallel/util/detail/chunk_size.hpp

```diff
@@ -152,9 +160,23 @@ namespace hpx { namespace parallel { namespace util { namespace detail
             {
                 std::size_t const cores = hpx::get_os_thread_count(exec);
                 if (count > 100*cores)
```

Comment:
The other function above multiplies the value returned by `hpx::get_os_thread_count()` by 2. Shouldn't that be done here as well?